modules/landing_zone/data.tf

```diff
@@ -1,4 +1,4 @@
-data "external" "landing_zone_output_file" {
-  depends_on = ["null_resource.landing_zone_apply"]
-  program    = ["sh", "${path.module}/scripts/output.sh", "${var.root_path}/output.json"]
+data "local_file" "landing_zone_output_file" {
+  depends_on = [null_resource.landing_zone_apply]
+  filename   = var.output_path
```

Comment:
please remove `output_path` as variable parameter. update the internal scripts to use `~/.terrahub/cache/output/[project_code]/output.json` that can be used across modules ;)